src/api/status/src/js/github-stats.js

```diff
@@ -0,0 +1,72 @@
+const { fetch } = require('@senecacdot/satellite');
+
+const fetchGitHubApi = (owner, repo, path) =>
+  fetch(`https://api.github.com/repos/${owner}/${repo}/${path}`, {
+    headers: { Accept: 'application/vnd.github.v3+json' },
+  });
+
+module.exports = async function getGitHubData(owner, repo) {
+  const returnedValue = {};
+
+  // get weekly commits for last year: https://docs.github.com/en/rest/reference/repos#get-the-weekly-commit-count
+  try {
+    const res = await fetchGitHubApi(owner, repo, 'stats/participation');
+    if (res.status >= 400) throw new Error(`[Error ${res.status}] - ${(await res.json()).message}`);
+
+    const participationResponse = await res.json();
+    returnedValue.weeklyCommits = {
+      commits: participationResponse.all[participationResponse.all.length - 1],
+    };
+    returnedValue.yearlyCommits = {
+      commits: participationResponse.all.reduce((a, b) => a + b),
+    };
```

Comment:
I want to separate those two since `weeklyCommits` contains weekly commit, and weekly linesAdded/linesRemoved.
`yearlyCommits` will contain yearly commit and if in the future we want to show yearly linesAdded/linesRemoved. `yearlyCommits` can be used.